.github/workflows/sync-docs.yml

```
@@ -12,60 +12,24 @@ jobs:
-          echo '${{ steps.files.outputs.payload }}' | jq '.'
-          
-          curl https://agentuity.ai/webhook/f61d5ce9d6ed85695cc992c55ccdc2a6 \
-            -X POST \
-            -H "Authorization: Bearer $AGENTUITY_TOKEN" \
-            -H "Content-Type: application/json" \
-            -d '${{ steps.files.outputs.payload }}'
\ No newline at end of file
+          set -euo pipefail
+          cat changed-files.txt | \
+            ./bin/build-payload.sh "${{ github.repository }}" incremental | \
+            ./bin/send-webhook.sh "https://agentuity.ai/webhook/f61d5ce9d6ed85695cc992c55ccdc2a6" "Bearer $AGENTUITY_TOKEN"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)